src/stylesheets/components/_sidebar.scss

```diff
@@ -0,0 +1,81 @@
+.sidebar {
+  position: fixed;
+  right: 0;
+  z-index: 1;
+
+  margin: 2rem 2rem 0;
+
+  &__user-image {
+    width: rem(36px);
+    height: rem(36px);
+
+    border-radius: 50%;
+
+    cursor: pointer;
+  }
+
+  &__nav-menu {
+    position: fixed;
+    top: 0;
+    right: 0;
+
+    justify-content: center;
+    width: 0;
+    height: 100%;
+    overflow: hidden;
+
+    background-color: map-get($brand-black, 'lighter');
+
+    backdrop-filter: blur(rem(82px));
+
+    transition: 0.5s;
+  }
+
+  &__nav-menu-active {
+    width: rem(264px);
+  }
+
+  &__nav-menu-items {
+    width: 100%;
+    margin: rem(32px) 0;
+    padding: 0;
+
+    list-style: none;
+  }
+
+  &__nav-menu-information {
+    display: flex;
+    align-items: center;
+    height: rem(36px);
+    margin: 0 rem(32px) rem(20px);
+  }
+
+  &__nav-menu-username {
+    color: map-get($brand-white, 'primary');
+    font-weight: map-get($font-weight, 'bolder');
+    font-size: map-get($font-sizes, 'xl');
+    line-height: rem(28px);
+  }
+
+  &__nav-menu-image {
+    margin-left: auto;
+  }
+
+  &__nav-menu-line {
```

Comment:
@rafayet-monon How about we add this as `border-bottom` to `&__nav-menu-information` so we don't need a separate class for the line, what do you think?